tools/src/main/scala/scala/scalanative/optimizer/inject/GCexternals.scala

```diff
@@ -0,0 +1,57 @@
+package scala.scalanative.optimizer.inject
+
+import scala.collection.mutable
+import scala.scalanative.nir._
+import scala.scalanative.optimizer.analysis.ClassHierarchy.Top
+import scala.scalanative.optimizer.{Inject, InjectCompanion}
+import scala.scalanative.tools.Config
+
+/**
+ * Created by lukaskellenberger on 08.03.17.
+ */
```

Comment:
We strongly discourage authorship comments in the code as they quickly get out of date.